### DISCUSSION TITLE
Do you recommend pinning organizational presets?

### DISCUSSION THREAD

**Discussion by sheldonhull** [renovatebot/renovate#29111] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### Please tell us more about your question or problem

- Can renovate submit PR's to update its own preset pinned version?
- If I want to have explicit control on the presets so I don't have to branch and can continue to iterate and then just tag on main... should I consider all repos adopting renovate use tag pinning? 

I couldn't find much discussion on if this is a good idea, if renovate creates PR's for this, or if it's more hassle than it's worth. Any general tips would be appreciated.

example:

```json
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["local>myproject/renovate-config#1.0.3"]
```

I publish a new version then it would create a new PR with 

```json
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["local>myproject/renovate-config#1.0.4"]
```

**Comment by rarkins** [comment, marked as answer]
I don't think most people would need to do this, because the "noise" it creates (a PR/commit in every repo each time there's a change to the central preset) likely isn't worth it for the "safety" it provides. Also if you set it to automatically merge the upgrades you probably break yourself anyway.

On the other hand, the more pinned your dependencies are (and this is in theory a dependency too), the more reproducible your software is, so I wouldn't call it a _bad_ idea.

This could perhaps be done with a custom regex manager, although "first class" support with its own manager would be nice too.

**Reply by HonkingGoose** [reply]
@rarkins do you want an issue to track the "add first class support for pinning org presets" then, or do you want to leave things as they are now?

**Reply by rarkins** [reply]
I thought there was an existing issue, but if not then we could have a low priority feature request to support pinned presets

**Reply by HonkingGoose** [reply]
See issue:

- #29946
